Formula/sail.rb

```diff
@@ -3,8 +3,9 @@ class Sail < Formula
 
   desc "CLI toolkit to provision and deploy WordPress applications to DigitalOcean"
   homepage "https://sailed.io"
-  url "https://files.pythonhosted.org/packages/60/4a/7d5a2481c233136d2234fed05569ed20ed97446e889f3996ee53e0132619/sailed.io-0.10.5.tar.gz"
-  sha256 "8dd2b00760389226470da22437daccfa2bc0f87134381d3742cc041ea2b39516"
+  url "https://files.pythonhosted.org/packages/a6/4e/e4d44426715fab487b4ef25b256097600091d3d97291899e53cec781cb8b/sailed.io-0.10.6.linux-x86_64.tar.gz"
```

Comment:
It looks like a wrong file was uploaded to pypi, it should be an archive with source code:

```
$ tar -ztvf sailed.io-0.10.6.linux-x86_64.tar.gz 
drwxrwxr-x  0 kovshenin kovshenin   0 Apr  7 12:20 ./
drwxrwxr-x  0 kovshenin kovshenin   0 Apr  7 12:20 ./home/
drwxrwxr-x  0 kovshenin kovshenin   0 Apr  7 12:20 ./home/kovshenin/
drwxrwxr-x  0 kovshenin kovshenin   0 Apr  7 12:20 ./home/kovshenin/Work/
drwxrwxr-x  0 kovshenin kovshenin   0 Apr  7 12:20 ./home/kovshenin/Work/sail-public/
drwxrwxr-x  0 kovshenin kovshenin   0 Apr  7 12:20 ./home/kovshenin/Work/sail-public/.env/
drwxrwxr-x  0 kovshenin kovshenin   0 Apr  7 12:20 ./home/kovshenin/Work/sail-public/.env/bin/
-rwxrwxr-x  0 kovshenin kovshenin 417 Apr  7 12:20 ./home/kovshenin/Work/sail-public/.env/bin/sail
drwxrwxr-x  0 kovshenin kovshenin   0 Apr  7 12:20 ./home/kovshenin/Work/sail-public/.env/lib/
drwxrwxr-x  0 kovshenin kovshenin   0 Apr  7 12:20 ./home/kovshenin/Work/sail-public/.env/lib/python3.9/
drwxrwxr-x  0 kovshenin kovshenin   0 Apr  7 12:20 ./home/kovshenin/Work/sail-public/.env/lib/python3.9/site-packages/
drwxrwxr-x  0 kovshenin kovshenin   0 Apr  7 12:20 ./home/kovshenin/Work/sail-public/.env/lib/python3.9/site-packages/sail/
-rw-rw-r--  0 kovshenin kovshenin 773 Feb  1 10:38 ./home/kovshenin/Work/sail-public/.env/lib/python3.9/site-packages/sail/__init__.py
-rw-rw-r--  0 kovshenin kovshenin  52 Sep  6  2021 ./home/kovshenin/Work/sail-public/.env/lib/python3.9/site-packages/sail/__main__.py
drwxrwxr-x  0 kovshenin kovshenin   0 Apr  7 12:20 ./home/kovshenin/Work/sail-public/.env/lib/python3.9/site-packages/sail/__pycache__/
-rw-rw-r--  0 kovshenin kovshenin 1186 Apr  7 12:20 ./home/kovshenin/Work/sail-public/.env/lib/python3.9/site-packages/sail/__pycache__/__init__.cpython-39.pyc
-rw-rw-r--  0 kovshenin kovshenin  234 Apr  7 12:20 ./home/kovshenin/Work/sail-public/.env/lib/python3.9/site-packages/sail/__pycache__/__main__.cpython-39.pyc
-rw-rw-r--  0 kovshenin kovshenin  517 Apr  7 12:20 ./home/kovshenin/Work/sail-public/.env/lib/python3.9/site-packages/sail/__pycache__/__version__.cpython-39.pyc
-rw-rw-r--  0 kovshenin kovshenin 5859 Apr  7 12:20 ./home/kovshenin/Work/sail-public/.env/lib/python3.9/site-packages/sail/__pycache__/backups.cpython-39.pyc
-rw-rw-r--  0 kovshenin kovshenin 12528 Apr  7 12:20 ./home/kovshenin/Work/sail-public/.env/lib/python3.9/site-packages/sail/__pycache__/blueprints.cpython-39.pyc
-rw-rw-r--  0 kovshenin kovshenin  4331 Apr  7 12:20 ./home/kovshenin/Work/sail-public/.env/lib/python3.9/site-packages/sail/__pycache__/cron.cpython-39.pyc
-rw-rw-r--  0 kovshenin kovshenin  7052 Apr  7 12:20 ./home/kovshenin/Work/sail-public/.env/lib/python3.9/site-packages/sail/__pycache__/database.cpython-39.pyc
-rw-rw-r--  0 kovshenin kovshenin  9293 Apr  7 12:20 ./home/kovshenin/Work/sail-public/.env/lib/python3.9/site-packages/sail/__pycache__/deploy.cpython-39.pyc
-rw-rw-r--  0 kovshenin kovshenin 12408 Apr  7 12:20 ./home/kovshenin/Work/sail-public/.env/lib/python3.9/site-packages/sail/__pycache__/domains.cpython-39.pyc
-rw-rw-r--  0 kovshenin kovshenin  4755 Apr  7 12:20 ./home/kovshenin/Work/sail-public/.env/lib/python3.9/site-packages/sail/__pycache__/misc.cpython-39.pyc
-rw-rw-r--  0 kovshenin kovshenin 17441 Apr  7 12:20 ./home/kovshenin/Work/sail-public/.env/lib/python3.9/site-packages/sail/__pycache__/profiling.cpython-39.pyc
-rw-rw-r--  0 kovshenin kovshenin 21368 Apr  7 12:20 ./home/kovshenin/Work/sail-public/.env/lib/python3.9/site-packages/sail/__pycache__/provision.cpython-39.pyc
-rw-rw-r--  0 kovshenin kovshenin  4922 Apr  7 12:20 ./home/kovshenin/Work/sail-public/.env/lib/python3.9/site-packages/sail/__pycache__/rebuild.cpython-39.pyc
-rw-rw-r--  0 kovshenin kovshenin  1768 Apr  7 12:20 ./home/kovshenin/Work/sail-public/.env/lib/python3.9/site-packages/sail/__pycache__/sftp.cpython-39.pyc
-rw-rw-r--  0 kovshenin kovshenin  5865 Apr  7 12:20 ./home/kovshenin/Work/sail-public/.env/lib/python3.9/site-packages/sail/__pycache__/ssh.cpython-39.pyc
-rw-rw-r--  0 kovshenin kovshenin  9929 Apr  7 12:20 ./home/kovshenin/Work/sail-public/.env/lib/python3.9/site-packages/sail/__pycache__/util.cpython-39.pyc
-rw-rw-r--  0 kovshenin kovshenin   314 Apr  7 12:04 ./home/kovshenin/Work/sail-public/.env/lib/python3.9/site-packages/sail/__version__.py
-rw-rw-r--  0 kovshenin kovshenin  7997 Dec 22 15:32 ./home/kovshenin/Work/sail-public/.env/lib/python3.9/site-packages/sail/backups.py
drwxrwxr-x  0 kovshenin kovshenin     0 Apr  7 12:20 ./home/kovshenin/Work/sail-public/.env/lib/python3.9/site-packages/sail/blueprints/
-rw-rw-r--  0 kovshenin kovshenin   557 Dec 27 08:04 ./home/kovshenin/Work/sail-public/.env/lib/python3.9/site-packages/sail/blueprints/default.yaml
-rw-rw-r--  0 kovshenin kovshenin   402 Sep 21  2021 ./home/kovshenin/Work/sail-public/.env/lib/python3.9/site-packages/sail/blueprints/fail2ban.yaml
-rw-rw-r--  0 kovshenin kovshenin   673 Sep 29  2021 ./home/kovshenin/Work/sail-public/.env/lib/python3.9/site-packages/sail/blueprints/gmail-dns.yaml
-rw-rw-r--  0 kovshenin kovshenin   800 Sep 22  2021 ./home/kovshenin/Work/sail-public/.env/lib/python3.9/site-packages/sail/blueprints/mailgun-dns.yaml
-rw-rw-r--  0 kovshenin kovshenin   607 Nov 18 07:40 ./home/kovshenin/Work/sail-public/.env/lib/python3.9/site-packages/sail/blueprints/mailgun.yaml
-rw-rw-r--  0 kovshenin kovshenin   967 Nov 16 07:35 ./home/kovshenin/Work/sail-public/.env/lib/python3.9/site-packages/sail/blueprints/postfix.yaml
-rw-rw-r--  0 kovshenin kovshenin  4088 Mar 10 11:25 ./home/kovshenin/Work/sail-public/.env/lib/python3.9/site-packages/sail/blueprints/sample.yaml
-rw-rw-r--  0 kovshenin kovshenin   300 Oct 15 10:47 ./home/kovshenin/Work/sail-public/.env/lib/python3.9/site-packages/sail/blueprints/site-verification.yaml
-rw-rw-r--  0 kovshenin kovshenin   292 Oct 15 10:18 ./home/kovshenin/Work/sail-public/.env/lib/python3.9/site-packages/sail/blueprints/test_apt.yaml
-rw-rw-r--  0 kovshenin kovshenin    96 Mar 10 11:25 ./home/kovshenin/Work/sail-public/.env/lib/python3.9/site-packages/sail/blueprints/test_commands.yaml
-rw-rw-r--  0 kovshenin kovshenin    97 Sep 17  2021 ./home/kovshenin/Work/sail-public/.env/lib/python3.9/site-packages/sail/blueprints/test_define.yaml
-rw-rw-r--  0 kovshenin kovshenin   473 Oct  7 09:23 ./home/kovshenin/Work/sail-public/.env/lib/python3.9/site-packages/sail/blueprints/test_dns.yaml
-rw-rw-r--  0 kovshenin kovshenin    93 Sep 21  2021 ./home/kovshenin/Work/sail-public/.env/lib/python3.9/site-packages/sail/blueprints/test_fail2ban.yaml
-rw-rw-r--  0 kovshenin kovshenin   166 Feb  1 12:12 ./home/kovshenin/Work/sail-public/.env/lib/python3.9/site-packages/sail/blueprints/test_files.yaml
-rw-rw-r--  0 kovshenin kovshenin   292 Sep 17  2021 ./home/kovshenin/Work/sail-public/.env/lib/python3.9/site-packages/sail/blueprints/test_options.yaml
-rw-rw-r--  0 kovshenin kovshenin   155 Sep 17  2021 ./home/kovshenin/Work/sail-public/.env/lib/python3.9/site-packages/sail/blueprints/test_plugins.yaml
-rw-rw-r--  0 kovshenin kovshenin   143 Sep 17  2021 ./home/kovshenin/Work/sail-public/.env/lib/python3.9/site-packages/sail/blueprints/test_themes.yaml
-rw-rw-r--  0 kovshenin kovshenin   640 Sep 21  2021 ./home/kovshenin/Work/sail-public/.env/lib/python3.9/site-packages/sail/blueprints/test_vars.yaml
-rw-rw-r--  0 kovshenin kovshenin 14668 Mar 10 11:25 ./home/kovshenin/Work/sail-public/.env/lib/python3.9/site-packages/sail/blueprints.py
-rw-rw-r--  0 kovshenin kovshenin  4594 Apr  7 08:01 ./home/kovshenin/Work/sail-public/.env/lib/python3.9/site-packages/sail/cron.py
-rw-rw-r--  0 kovshenin kovshenin  7784 Mar  2 19:59 ./home/kovshenin/Work/sail-public/.env/lib/python3.9/site-packages/sail/database.py
-rw-rw-r--  0 kovshenin kovshenin 11494 Mar 22 08:08 ./home/kovshenin/Work/sail-public/.env/lib/python3.9/site-packages/sail/deploy.py
-rw-rw-r--  0 kovshenin kovshenin 14163 Jan 29 11:05 ./home/kovshenin/Work/sail-public/.env/lib/python3.9/site-packages/sail/domains.py
-rw-rw-r--  0 kovshenin kovshenin  4989 Feb  1 16:16 ./home/kovshenin/Work/sail-public/.env/lib/python3.9/site-packages/sail/misc.py
drwxrwxr-x  0 kovshenin kovshenin     0 Apr  7 12:20 ./home/kovshenin/Work/sail-public/.env/lib/python3.9/site-packages/sail/premium/
-rw-rw-r--  0 kovshenin kovshenin  3502 Dec 22 15:32 ./home/kovshenin/Work/sail-public/.env/lib/python3.9/site-packages/sail/premium/__init__.py
drwxrwxr-x  0 kovshenin kovshenin     0 Apr  7 12:20 ./home/kovshenin/Work/sail-public/.env/lib/python3.9/site-packages/sail/premium/__pycache__/
-rw-rw-r--  0 kovshenin kovshenin  3516 Apr  7 12:20 ./home/kovshenin/Work/sail-public/.env/lib/python3.9/site-packages/sail/premium/__pycache__/__init__.cpython-39.pyc
-rw-rw-r--  0 kovshenin kovshenin  5829 Apr  7 12:20 ./home/kovshenin/Work/sail-public/.env/lib/python3.9/site-packages/sail/premium/__pycache__/backups.cpython-39.pyc
-rw-rw-r--  0 kovshenin kovshenin  8877 Apr  7 12:20 ./home/kovshenin/Work/sail-public/.env/lib/python3.9/site-packages/sail/premium/__pycache__/monitor.cpython-39.pyc
-rw-rw-r--  0 kovshenin kovshenin  6715 Dec 22 15:32 ./home/kovshenin/Work/sail-public/.env/lib/python3.9/site-packages/sail/premium/backups.py
-rw-rw-r--  0 kovshenin kovshenin  9683 Mar 11 07:44 ./home/kovshenin/Work/sail-public/.env/lib/python3.9/site-packages/sail/premium/monitor.py
-rw-rw-r--  0 kovshenin kovshenin 21793 Mar 23 08:17 ./home/kovshenin/Work/sail-public/.env/lib/python3.9/site-packages/sail/profiling.py
-rw-rw-r--  0 kovshenin kovshenin 24970 Mar 29 14:39 ./home/kovshenin/Work/sail-public/.env/lib/python3.9/site-packages/sail/provision.py
-rw-rw-r--  0 kovshenin kovshenin  5260 Jan 10 12:49 ./home/kovshenin/Work/sail-public/.env/lib/python3.9/site-packages/sail/rebuild.py
-rw-rw-r--  0 kovshenin kovshenin  1353 Feb  1 11:36 ./home/kovshenin/Work/sail-public/.env/lib/python3.9/site-packages/sail/sftp.py
-rw-rw-r--  0 kovshenin kovshenin  5804 Mar 22 08:17 ./home/kovshenin/Work/sail-public/.env/lib/python3.9/site-packages/sail/ssh.py
drwxrwxr-x  0 kovshenin kovshenin     0 Apr  7 12:20 ./home/kovshenin/Work/sail-public/.env/lib/python3.9/site-packages/sail/templates/
-rw-rw-r--  0 kovshenin kovshenin    43 Mar 10 11:25 ./home/kovshenin/Work/sail-public/.env/lib/python3.9/site-packages/sail/templates/certbot.deploy.sh
-rw-rw-r--  0 kovshenin kovshenin   702 Mar 23 08:42 ./home/kovshenin/Work/sail-public/.env/lib/python3.9/site-packages/sail/templates/cloud-config.yaml
drwxrwxr-x  0 kovshenin kovshenin     0 Apr  7 12:20 ./home/kovshenin/Work/sail-public/.env/lib/python3.9/site-packages/sail/templates/fail2ban/
-rw-rw-r--  0 kovshenin kovshenin   892 Nov  9 14:00 ./home/kovshenin/Work/sail-public/.env/lib/python3.9/site-packages/sail/templates/fail2ban/0-sail-auth-syslog.php
-rw-rw-r--  0 kovshenin kovshenin   664 Nov  9 14:00 ./home/kovshenin/Work/sail-public/.env/lib/python3.9/site-packages/sail/templates/fail2ban/jail.local
-rw-rw-r--  0 kovshenin kovshenin   330 Nov  9 14:00 ./home/kovshenin/Work/sail-public/.env/lib/python3.9/site-packages/sail/templates/fail2ban/nginx-deny.conf
-rw-rw-r--  0 kovshenin kovshenin    48 Nov  9 14:00 ./home/kovshenin/Work/sail-public/.env/lib/python3.9/site-packages/sail/templates/fail2ban/wordpress-auth.conf
-rw-rw-r--  0 kovshenin kovshenin    44 Nov  9 14:00 ./home/kovshenin/Work/sail-public/.env/lib/python3.9/site-packages/sail/templates/fail2ban/wordpress-pingback.conf
-rw-rw-r--  0 kovshenin kovshenin   199 Jan 21 10:09 ./home/kovshenin/Work/sail-public/.env/lib/python3.9/site-packages/sail/templates/logrotate.conf
-rw-rw-r--  0 kovshenin kovshenin    95 Feb  1 16:32 ./home/kovshenin/Work/sail-public/.env/lib/python3.9/site-packages/sail/templates/mariadb.cnf
-rw-rw-r--  0 kovshenin kovshenin   887 Nov  9 14:00 ./home/kovshenin/Work/sail-public/.env/lib/python3.9/site-packages/sail/templates/nginx.certbot.conf
-rw-rw-r--  0 kovshenin kovshenin  1599 Jan  8 11:31 ./home/kovshenin/Work/sail-public/.env/lib/python3.9/site-packages/sail/templates/nginx.main.conf
-rw-rw-r--  0 kovshenin kovshenin   489 Dec 22 15:32 ./home/kovshenin/Work/sail-public/.env/lib/python3.9/site-packages/sail/templates/nginx.main.premium.conf
-rw-rw-r--  0 kovshenin kovshenin  1452 Dec 22 15:32 ./home/kovshenin/Work/sail-public/.env/lib/python3.9/site-packages/sail/templates/nginx.premium.conf
-rw-rw-r--  0 kovshenin kovshenin  1096 Dec 17 08:34 ./home/kovshenin/Work/sail-public/.env/lib/python3.9/site-packages/sail/templates/nginx.server.conf
-rw-rw-r--  0 kovshenin kovshenin   783 Jan 21 09:28 ./home/kovshenin/Work/sail-public/.env/lib/python3.9/site-packages/sail/templates/nginx.shared.conf
-rw-rw-r--  0 kovshenin kovshenin 73032 Dec 15 20:14 ./home/kovshenin/Work/sail-public/.env/lib/python3.9/site-packages/sail/templates/php.ini
drwxrwxr-x  0 kovshenin kovshenin     0 Apr  7 12:20 ./home/kovshenin/Work/sail-public/.env/lib/python3.9/site-packages/sail/templates/postfix/
-rw-rw-r--  0 kovshenin kovshenin   377 Nov 15 09:45 ./home/kovshenin/Work/sail-public/.env/lib/python3.9/site-packages/sail/templates/postfix/1-sail-mail-from.php
-rw-rw-r--  0 kovshenin kovshenin  1070 Nov 16 07:24 ./home/kovshenin/Work/sail-public/.env/lib/python3.9/site-packages/sail/templates/postfix/main.cf
-rw-rw-r--  0 kovshenin kovshenin  2373 Dec 22 15:32 ./home/kovshenin/Work/sail-public/.env/lib/python3.9/site-packages/sail/templates/premium.php
-rw-rw-r--  0 kovshenin kovshenin  1796 Dec 22 15:32 ./home/kovshenin/Work/sail-public/.env/lib/python3.9/site-packages/sail/templates/prepend.php
drwxrwxr-x  0 kovshenin kovshenin     0 Apr  7 12:20 ./home/kovshenin/Work/sail-public/.env/lib/python3.9/site-packages/sail/tests/
-rw-rw-r--  0 kovshenin kovshenin    15 Sep  6  2021 ./home/kovshenin/Work/sail-public/.env/lib/python3.9/site-packages/sail/tests/__init__.py
drwxrwxr-x  0 kovshenin kovshenin     0 Apr  7 12:20 ./home/kovshenin/Work/sail-public/.env/lib/python3.9/site-packages/sail/tests/__pycache__/
-rw-rw-r--  0 kovshenin kovshenin   179 Apr  7 12:20 ./home/kovshenin/Work/sail-public/.env/lib/python3.9/site-packages/sail/tests/__pycache__/__init__.cpython-39.pyc
-rw-rw-r--  0 kovshenin kovshenin  2219 Apr  7 12:20 ./home/kovshenin/Work/sail-public/.env/lib/python3.9/site-packages/sail/tests/__pycache__/end2end-noinit.cpython-39.pyc
-rw-rw-r--  0 kovshenin kovshenin 20099 Apr  7 12:20 ./home/kovshenin/Work/sail-public/.env/lib/python3.9/site-packages/sail/tests/__pycache__/end2end.cpython-39.pyc
-rw-rw-r--  0 kovshenin kovshenin  2997 Apr  7 12:20 ./home/kovshenin/Work/sail-public/.env/lib/python3.9/site-packages/sail/tests/__pycache__/test_destroy.cpython-39.pyc
-rw-rw-r--  0 kovshenin kovshenin   962 Apr  7 12:20 ./home/kovshenin/Work/sail-public/.env/lib/python3.9/site-packages/sail/tests/__pycache__/test_version.cpython-39.pyc
-rw-rw-r--  0 kovshenin kovshenin  1605 Mar 10 10:19 ./home/kovshenin/Work/sail-public/.env/lib/python3.9/site-packages/sail/tests/end2end-noinit.py
-rw-rw-r--  0 kovshenin kovshenin 26041 Mar 10 11:25 ./home/kovshenin/Work/sail-public/.env/lib/python3.9/site-packages/sail/tests/end2end.py
-rw-rw-r--  0 kovshenin kovshenin  2989 Nov 16 09:22 ./home/kovshenin/Work/sail-public/.env/lib/python3.9/site-packages/sail/tests/test_destroy.py
-rw-rw-r--  0 kovshenin kovshenin   500 Sep  6  2021 ./home/kovshenin/Work/sail-public/.env/lib/python3.9/site-packages/sail/tests/test_version.py
-rw-rw-r--  0 kovshenin kovshenin  8473 Mar 29 14:39 ./home/kovshenin/Work/sail-public/.env/lib/python3.9/site-packages/sail/util.py
drwxrwxr-x  0 kovshenin kovshenin     0 Apr  7 12:20 ./home/kovshenin/Work/sail-public/.env/lib/python3.9/site-packages/sailed.io-0.10.6.egg-info/
-rw-rw-r--  0 kovshenin kovshenin   280 Apr  7 12:20 ./home/kovshenin/Work/sail-public/.env/lib/python3.9/site-packages/sailed.io-0.10.6.egg-info/PKG-INFO
-rw-rw-r--  0 kovshenin kovshenin  1977 Apr  7 12:20 ./home/kovshenin/Work/sail-public/.env/lib/python3.9/site-packages/sailed.io-0.10.6.egg-info/SOURCES.txt
-rw-rw-r--  0 kovshenin kovshenin     1 Apr  7 12:20 ./home/kovshenin/Work/sail-public/.env/lib/python3.9/site-packages/sailed.io-0.10.6.egg-info/dependency_links.txt
-rw-rw-r--  0 kovshenin kovshenin    35 Apr  7 12:20 ./home/kovshenin/Work/sail-public/.env/lib/python3.9/site-packages/sailed.io-0.10.6.egg-info/entry_points.txt
-rw-rw-r--  0 kovshenin kovshenin   149 Apr  7 12:20 ./home/kovshenin/Work/sail-public/.env/lib/python3.9/site-packages/sailed.io-0.10.6.egg-info/requires.txt
-rw-rw-r--  0 kovshenin kovshenin     5 Apr  7 12:20 ./home/kovshenin/Work/sail-public/.env/lib/python3.9/site-packages/sailed.io-0.10.6.egg-info/top_level.txt
```

Reply:
Good catch! Fixed and force pushed.